xml/xen_virtualization_vhost.xml

```diff
@@ -156,10 +156,23 @@
     <step>
      <para>
       At &dom0;, type <command>xl info</command> to view the amount of memory
-      that is available on the machine. &dom0;'s current memory allocation can
+      that is available on the machine. &dom0;'s cur rent memory allocation can
       be determined with the <command>xl list</command> command.
      </para>
     </step>
+    <step>
+     <para>
+      Edit <filename>/etc/default/grub</filename> and adjust the
+      <option>GRUB_CMDLINE_XEN</option> option so that it includes
+      <literal>dom0_mem=<replaceable>MEM_AMOUNT</replaceable></literal>.
+      Replace <replaceable>MEM_AMOUNT</replaceable> with the maximum amount of
+      memory to allocate to &dom0;. Add <command>K</command>,
+      <command>M</command>, or <command>G</command>, to specify the size unit.
+      For example:
+     </para>
+<screen>GRUB_CMDLINE_XEN="dom0_mem=2G"</screen>
+    </step>
```

Comment:
You added the step describing use of /etc/default/grub, but didn't remove the yast steps. IIUC those should be removed right?

Reply:
a new commit fixes that, i forgot to push 1 change...

Reply:
Looks good now. Thanks!